### PR TITLE
Enable Partial PlaidML test cases

### PR DESCRIPTION
**Pull request by chencha3**
Enable a few more PlaidML test cases, for those failed are now temporarily excluded in lit.local.config

### CODE REVIEW

File: test/PlaidML/OpTest.DotF16.mlir [hunk 1]
```diff
@@ -2,10 +2,10 @@
 // RUN:                                       --runner mlir-cpu-runner -e main \
 // RUN:                                       --shared-libs=%mlir_runner_utils \
 // RUN:                                       --entry-point-result=void | FileCheck %s
-// RUN-GPU: %python_executable %imex_runner -i %s --pass-pipeline-file=%p/linalg-to-llvm.pp \
-// RUN-GPU:                                        --runner mlir-cpu-runner -e main \
-// RUN-GPU:                                        --entry-point-result=void \
-// RUN-GPU:                                        --shared-libs=%mlir_runner_utils,%sycl_runtime | FileCheck %s
+// RUN: %python_executable %imex_runner -i %s --pass-pipeline-file=%p/linalg-to-llvm.pp \
```
Comment (nbpatel) [inline test/PlaidML/OpTest.DotF16.mlir:5]: Keep the GPU disabled for now.....CI does not have the setup yet

File: build_tools/llvm_version.txt [hunk 2]
```diff
@@ -1 +1 @@
-2a827e4a988b614bc6f70abe00308ceeb50dcd0a
+cb2cb2d201811d95d4f35337817faea05654d8f0
```
Comment (nbpatel) [inline build_tools/llvm_version.txt:1]: remove this..we need to change llvm version in separate PR

File: test/PlaidML/lit.local.cfg [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+                  'OpTest.LogicalAnd_mixed.mlir',
+                  'OpTest.LogicalNot_float.mlir',
+                  'OpTest.LogicalOr_float.mlir',
+                  'OpTest.LogicalOr_uint64.mlir',
+                  'OpTest.LogicalAnd_uint64.mlir',
+                  'OpTest.LogicalNot_int32.mlir',
+                  'OpTest.LogicalOr_int32.mlir',
+                  'OpTest.MnistCnn.mlir',
+                  'OpTest.SinH.mlir',
+                  'OpTest.Sum.mlir',
```
Comment (nbpatel) [inline test/PlaidML/lit.local.cfg:14]: remove sum from here...sum works now

File: test/PlaidML/OpTest.MaxPool1D.mlir [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+// RUN-GPU:                                       --entry-point-result=void \
+// RUN-GPU:                                       --shared-libs=%mlir_runner_utils,%sycl_runtime | FileCheck %s
+#map0 = affine_map<(d0, d1) -> (d1)>
+#map1 = affine_map<(d0, d1) -> (d0)>
+module @max_pool_1d {
+func.func @main() {
+    %0= arith.constant dense<[1, 2, 3]>:tensor<3xi64>
+    %1 = call @test(%0) : (tensor<3xi64>) -> tensor<1xi64>
+    %unranked = tensor.cast %1 : tensor<1xi64>to tensor<*xi64>
+<<<<<<< HEAD
```
Comment (nbpatel) [inline test/PlaidML/OpTest.MaxPool1D.mlir:16]: merge issues

File: test/PlaidML/lit.local.cfg (hunk 3 above)
Comment (chencha3) [reply test/PlaidML/lit.local.cfg:14]: addressed these issues.

**Review by nbpatel — APPROVED** [review]
LGTM

**Pull request merged**